src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -472,6 +472,8 @@ public enum SslProtocols
         Tls = 192,
         Tls11 = 768,
         Tls12 = 3072,
+        [Obsolete("This value has been deprecated.  It is no longer supported. http://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
Thanks!

>  I assume these are all okay: #13771

It's hard to tell. We found about the Xml issue via our AppCompat lab.